src/components/common/Pagination.tsx

```diff
@@ -37,36 +37,40 @@ export default function Pagination({
   const pageNumbers = range(start, end);
 
   return (
-    <div className="flex items-center justify-center gap-3 md:gap-4 lg:gap-5">
+    <div className="flex items-center justify-center gap-20 py-12">
       {/* 왼쪽 화살표 */}
       {totalPages > 7 && (
         <button type="button" onClick={() => handleClick(1)}>
           <img
             src={currentPage === 1 ? ArrowLeftGray : ArrowLeft}
             alt="이전"
-            className="size-5"
+            className="size-20"
           />
         </button>
       )}
+
       {/* 페이지 버튼들 */}
-      <div className="flex gap-2 sm:gap-1">
+      <div className="flex gap-2">
```

Comment:
```suggestion
      <div className="flex gap-4 md:gap-2">
```
❗ mobile일때 gap 크기가 달라요!

Reply:
감사합니다 수정했씁니다..자꾸 제가 보기에 gap이 안 맞는것같아서 일단 지정한거 계속 안 바꿨네요... 감사합니당